content/help/ops/perf/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Performance Tuning
+description: Istio performance, sizing and tuning guide.
+weight: 90
+force_inline_toc: true
+---
+## Istio Performance Tuning
+
+Istio's goal is to have a performant out of the box configuration.
+However it is difficult to create a configuration that is performant for all use cases.
+
+The following section documents a few configuration knobs that affect performance.
+
+| Knob | Default | Symptom | Description |
+| --- | --- | --- | --- |
+| [`Sidecar resource`](/docs/reference/config/networking/v1alpha3/sidecar) | Disabled | High memory usage by sidecars and high CPU by Pilot | Enable namespace isolation to get a significant reduction in sidecar memory usage and pilot CPU usage. |
+| `global.proxy.concurrency` | `2` | Sidecar resource utilization and latency.| This values should be at least equal to the number of CPU core allocated to the proxy. |
+| `pilot.keepaliveMaxServerConnectionAge` | `30m` | Uneven load distribution across Pilot replicas. | Reduce this number if you want quicker balancing of load at the cost of increased system churn. |
+| `pilot.traceSampling` | `1.0` |  Latency and throughput impact due to tracing. | This can be reduced further from the default. If set to 0.0%, use `x-client-trace-id` header to trace specific requests. |
+| `global.disablePolicyChecks` | `true` | Tail latency impact due to policy. | Enable policy checks *only* if you are using rate limiting or another istio-policy feature.|
```

Comment:
Is there a page that can be linked to that defines what istio-policy features exist?

Reply:
There are two possible links in the conceptual and reference docs: 
- https://preliminary.istio.io/docs/concepts/policies-and-telemetry/
- https://preliminary.istio.io/docs/reference/config/istio.authentication.v1alpha1/
That said, I recommend pointing to the rate limiting task since that is the feature mentioned:
https://preliminary.istio.io/docs/tasks/policy-enforcement/rate-limiting/